wcc-contentful-app/spec/dummy/app/helpers/section_helper_spec.rb

```diff
@@ -50,6 +50,82 @@
         expect(html_to_render).to include("{: .this }")
       end
     end
+
+    context 'when class syntax is used but without a dot on the class {: this }' do
+      it 'returns the {: this } within the html to be rendered' do
+        markdown_string = "Just some content {: this }"
+        html_to_render =
+          helper.markdown(markdown_string)
+
+        expect(html_to_render).to include("{: this }")
+      end
+    end
+
+    context 'when only class syntax is passed {: .this } without link' do
+      it 'returns the {: .this } within the html to be rendered' do
+        markdown_string = "Just some content {: .this }"
+        html_to_render =
+          helper.markdown(markdown_string)
+
+        expect(html_to_render).to include("{: .this }")
+      end
+    end
+
+    context 'When given: [links with a newline]\n(https://www.test.com){: .newline }' do
+      it 'will render the string as a mashed up mess because of the \n' do
+        markdown_string =
+          'some before text [links with a newline]\n(https://www.test.com){: .newline } and some after text'
```

Comment:
In Ruby strings, the single quote does not translate `\n` into an actual newline character.  Instead it's preserved as a literal `\n`.  To get it to become a newline character you should use double quotes.  I tested this with redcarpet, it does in fact recognize the following:

```ruby
irb(main):002:0> WCC::Contentful::App::SectionHelper.markdown('some before text [links with a newline]\n(https://www.test.com){: .newline } and some after text')
=> "<p>some before text [links with a newline]\\n(<a href=\"https://www.test.com)%7B:\" rel=\"nofollow\" target=\"_blank\">https://www.test.com){:</a> .newline } and some after text</p>\n"
irb(main):003:0> WCC::Contentful::App::SectionHelper.markdown("some before text [links with a newline]\n(https://www.test.com){: .newline } and some after text")
=> "<p>some before text <a href=\"https://www.test.com\" rel=\"nofollow\" target=\"_blank\">links with a newline</a>{: .newline } and some after text</p>\n"
```